src/main/java/com/novauc/MessageController.java

```diff
@@ -0,0 +1,62 @@
+package com.novauc;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import javax.servlet.http.HttpSession;
+import java.util.ArrayList;
+import java.util.Iterator;
+
+/**
+ * Created by jerieshasmith on 3/13/17.
+ */
+@Controller
+
+public class MessageController {
+    @RequestMapping(path = "/", method = RequestMethod.GET)
+    public String home(Model model, HttpSession session) {
+        model.addAttribute("name", session.getAttribute("userName"));
+        return "home";
```

Comment:
Your messages do not display.  Please fix that and the delete method before resubmitting.  REMEMBER anything that needs to show up on html MUST be added to the model first.